src/webview/css/static/app.css

```diff
@@ -15,7 +16,9 @@ body * {
 
 body footer {
   position: sticky;
-  bottom: 0;
+  bottom: 2px;
```

Comment:
Looks much better, only `bottom: 2px` creates a gap through which you can see the diff while scrolling, I find it distracting... 

<img width="592" alt="Screen Shot 2023-01-20 at 18 06 05" src="https://user-images.githubusercontent.com/807315/213774600-97dfbbbe-a854-4c64-ae25-8c80b3a7ebbe.png">

Reply:
Thanks for checking! I will take a look :)